datasets/new-dataset-using-llm/README.md

```diff
@@ -0,0 +1,79 @@
+# Synthetic Data Generation
+## Overall
+Leveraging large language models(LLM) is a crucial technique in synthetic data generation, capable of producing high-quality, diverse, and privacy-compliant datasets. This approach can be applied in various scenarios, including training machine learning models, fine-tuning different language models, and conducting evaluation and testing, among others.
+
+> Note: If you are using an LLM to generate a dataset for fine-tuning another model, please ensure you adhere to the relevant licenses.
+
+## Benefits of synthetic data
+- Privacy Protection
+
+  Synthetic data mimics real data without including sensitive information, making it perfect for fields with strict privacy rules.
+
+- Structured and Easy to Use
+
+  Synthetic data is often well-structured and can come pre-labeled, simplifying data preparation for machine learning.
+
+- Data Augmentation
+
+  It can fill gaps in sparse datasets, making them richer and more diverse, which is useful for balancing datasets.
+
+- Unlimited Data Generation
+
+  You can generate synthetic data on-demand and at a large scale, providing a cost-effective way to get more training data.
+
+- Bias Reduction
+
+  Synthetic data can help reduce bias in AI models by balancing out biased datasets, leading to fairer AI systems.
+
+## Generate a synthetic dataset
+**Step 1: Setup the Python Environment**
+
+Install the required Python packages:
+```sh
+pip install -r requirements.txt
+```
+
+**Step 2: Prepare Your Inference Endpoint**
```

Comment:
What is the by default option? should be LLM?

Reply:
Changed the default option to be azure openai